src/components/bouquets/BouquetCard.vue

```diff
@@ -44,49 +46,47 @@ const { themeColors } = useThemeOptions(theme)
 </script>
 
 <template>
-  <article class="fr-my-3w fr-p-3w border border-default-grey fr-enlarge-link">
+  <article
+    class="fr-my-1w fr-px-3w fr-py-2w border border-default-grey fr-enlarge-link"
```

Comment:
24px partout sur la maquette

```suggestion
    class="fr-my-1w fr-p-3w border border-default-grey fr-enlarge-link"
```

J'avais pas fait attention que ça restait pareil en mobile… Ça me semble être beaucoup

Reply:
Avec ce résultat j'ai un espacement plus grand en haut que sur les côtés, ce qui ne fait pas trop harmonieux. C'est, il me semble, plus agréable quand on distingue une carré d'espacement dans les angles